modules/rules_robolectric/4.11.1-1.0.0/MODULE.bazel

```diff
@@ -0,0 +1,37 @@
+"Bazel dependencies"
+
+module(
+    name = "rules_robolectric",
+    # Note: the publish-to-BCR app will patch this line to stamp the version being published.
+    version = "4.11.1-1.0.0",
```

Comment:
hmm, hang on. are you aware that this version compares _older_ than `4.11.1`?

Reply:
https://github.com/bazelbuild/bazel-central-registry/pull/1595/files#diff-5a922c15884afe67344db999c204f6e5c06084ad56f71b702a866c3502642e71R19 oof no I was not aware. Good call out.

Do you have any recommendations for how to approach this one?

Reply:
why is this not just 4.11.2? is the version name tied to something else?

you could name it 4.11.1.bazel.1 or some such, in that case.

Reply:
We are trying to track the upstream Robolectric releases so that's it's clear which version you should be using while also leaving room for incremental changed to the rule set between Robolectric releases.

Context: https://github.com/robolectric/robolectric-bazel/issues/95